util/mockdata/mockdata.go

```diff
@@ -0,0 +1,22 @@
+package mockdata
+
+import (
+	"github.com/goombaio/namegenerator"
+	"math/rand"
+)
+
+const (
+	MockDBDSN  = "mockDB"
+	DriverName = "postgres"
+)
+
+func genSeed() (seed int64) {
```

Comment:
Why did you declare this function as private?

Reply:
I think it's should private because it's only use in this package, but I change to public for the future feature use this function